lib/wanda/results.ex

```diff
@@ -69,6 +65,26 @@ defmodule Wanda.Results do
     |> Repo.one()
   end
 
+  @doc """
+  Marks a previously started execution as completed
+  """
+  @spec complete_execution_result!(Result.t()) :: ExecutionResult.t()
+  def complete_execution_result!(%Result{execution_id: execution_id} = result) do
+    case get_execution_result!(execution_id) do
+      %ExecutionResult{
+        status: :running
+      } = execution ->
+        execution
+        |> ExecutionResult.complete(result)
+        |> Repo.update!()
+
+      %ExecutionResult{
+        status: :completed
+      } = execution ->
+        execution
```

Comment:
agreed on return ok or error if it was already completed

Reply:
adjusted here https://github.com/trento-project/wanda/pull/60/commits/a272e9ce8df9bd3dcb4de5e0ee618ffa3e67472e